core/src/main/java/org/frankframework/soap/SoapWrapper.java

```diff
@@ -278,13 +278,15 @@ public Message putInEnvelope(Message message, String encodingStyleUri, String ta
 				if (separatorPos < 1) {
 					namespaceClause.append(" xmlns=\"").append(namespaceDef).append("\"");
 				} else {
-					namespaceClause.append(" xmlns:" + namespaceDef.substring(0, separatorPos) + "=\"" + namespaceDef.substring(separatorPos + 1) + "\"");
+					namespaceClause.append(" xmlns:").append(namespaceDef.substring(0, separatorPos)).append("=\"").append(namespaceDef.substring(separatorPos + 1)).append("\"");
```

Comment:
When making this optimization, you can also get rid of the `.substring(...)`, you can pass these directly to the `.append(...)` method.

Reply:
Done.